File: IP-Lab1/Biblioteca.cpp
///Biblioteca.cpp
#include "Biblioteca.h"
#include <iostream>

Biblioteca::Biblioteca()
{ }

void Biblioteca::AddCarte(Carte c)
{
	this->carti.push_back(c);
}

void Biblioteca::Print()
{
	for (auto i : carti)
	{
		i.Print();
	}
}

File: IP-Lab1/Lab1Java/src/Biblioteca.java
import java.util.ArrayList;
import java.util.List;

public class Biblioteca {
    private List<Carte> carti;

    public Biblioteca()
    {
        carti=new ArrayList<>();
    };

    public void addCarte(Carte c)
    {
        this.carti.add(c);
    }

    public void print()
    {
        for (var c:carti)
        {
            c.Print();
        }
    }

}
